examples/asr/conf/speech_multitask/fast-conformer_aed.yaml

```diff
@@ -0,0 +1,280 @@
+# It contains the default values for training an autoregressive FastConformer-Transformer ST model with sub-word encoding.
+
+# Architecture and training config:
+# Default learning parameters in this config are set for effective batch size of 2K. To train it with smaller effective
+# batch sizes, you may need to re-tune the learning parameters or use higher accumulate_grad_batches.
+# Here are the recommended configs for different variants of FastConformer-Transformer, other parameters are the same as in this config file.
+# One extra (linear projection) layer is added between FastConformer encoder and Transformer decoder if they have different hidden sizes
+# It is recommended to initialize FastConformer with ASR pre-trained encoder for better accuracy and faster convergence
+
+name: "FastConformer-Transformer-MultiTask"
+
+# Initialize model encoder with pre-trained ASR FastConformer encoder for faster convergence and improved accuracy
+init_from_nemo_model:
+  model0:
+    path: ???
+    include: ["preprocessor", "encoder"]
+
+model:
+  _target_: nemo.collections.asr.models.EncDecMultiTaskModel
+  sample_rate: 16000
+  label_smoothing: 0.0
+  context_len_for_AR_decoding: 5
+  log_prediction: true # enables logging sample predictions in the output during training
+
+  # Important ! Set the prompt format to the class you need
+  prompt_format: ???   # Options supported: ["canary"]
```

Comment:
If only one option is supported, should this be mandated?

Reply:
Following @titu1994 suggestions we wanted to make the "multi task" class as independent from the concept of "canary" as possible, which guided this design choice. CC @titu1994 @krishnacpuvvada

Reply:
yes, two reasons 1. what piotr mentioned. and (2) it is an important parameter that we want the user to pay attention to it and not become complacent with time.

Reply:
Agreed, don't make canary default. The class should be agnostic to prompt format, we will play around with it in the future.